test/protocol/disconnect_dap_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative '../support/protocol_test_case'
+
+module DEBUGGER__
+
+  class DisconnectDAPTest < ProtocolTestCase
+    PROGRAM = <<~RUBY
+       1| module Foo
+       2|   class Bar
+       3|     def self.a
+       4|       "hello"
+       5|     end
+       6|   end
+       7|   loop do
+       8|     b = 1
+       9|   end
+      10|   Bar.a
+      11|   bar = Bar.new
+      12| end
+    RUBY
+
+    def test_disconnect_without_terminateDebuggee_keeps_debuggee_alive
+      run_protocol_scenario PROGRAM, cdp: false do
+        req_dap_disconnect(terminate_debuggee: false)
+        attach_to_dap_server
```

Comment:
If you want to use `attach_to_dap_server`, please move please move it into API block and update document.

https://github.com/ruby/debug/blob/master/test/support/protocol_test_case.rb#L41

Reply:
👍 done